AnsibleTestCRD.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: test.openstack.org/v1beta1
+kind: AnsibleTest
+metadata:
+  name: rhel-ai-validation-ansibletest
+  namespace: openstack
+spec:
+  ansibleGitRepo: https://github.com/openstack-k8s-operators/rhelai-validation.git
```

Comment:
You can't test this as-is because we don't yet have compatible content on the default branch of this repo. Push to your own fork's main branch to test.

Reply:
I'm going to mark this as resolved, otherwise it is preventing the merging of this PR.